run.py

```diff
@@ -16,7 +16,7 @@
         ">>> ", ">> ", "> ",
         ">>>", ">>", ">"
     ),  # Order matters (and so do commas)
-    activity=Game(name="Help: bot.help()"),
+    activity=Game(name="with snekky sneks"),
```

Comment:
Probably a bit user-unfriendly, but it's fine in this case. :P

Reply:
still mentionable /shrug